cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -166,7 +172,8 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run existing copyrigh
             scanner.scanOrtResult(
                 ortResultFile = ortFile,
                 outputDirectory = absoluteNativeOutputDir,
-                downloadDirectory = absoluteDownloadDir ?: absoluteOutputDir.resolve("downloads")
+                downloadDirectory = absoluteDownloadDir ?: absoluteOutputDir.resolve("downloads"),
+                ignoreExcluded = skipExcluded
```

Comment:
The function parameter is also called `skipExcluded`.

Reply:
Done.